audobject/core/api.py

```diff
@@ -26,7 +26,7 @@ def from_dict(  # noqa: D417
     auto_install: bool = False,
     override_args: dict[str, object] | None = None,
     **kwargs,
-) -> Object:
+) -> "Object":
```

Comment:
Why have you reverted this as well?

Reply:
As it was the original state in the current main branch.

Reply:
Yes, but this should also be no longer required now.

Reply:
```suggestion
) -> Object:
```

Reply:
There were more isntances of the quoted type hint  `-> "Object"`. All of these have been changed to `-> Object` in 
efbbb4e.